XenAdmin/Program.cs

```diff
@@ -429,12 +429,12 @@ private static void ConnectPipe()
 
                     if (firstArgType == ArgType.Passwords)
                     {
-                        log.Error("Refusing to accept passwords request down pipe.  Use XenCenterMain.exe directly");
+                        log.Error("Refusing to accept passwords request down pipe.  Use CitrixHypervisorManagementMain.exe directly");

```

Comment:
`log.ErrorFormat("Refusing to accept passwords request down pipe. Use {0}Main.exe directly", BrandManager.BRAND_CONSOLE.Replace(" ",""));`
Same below.